packages/primitives/src/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react'
+import { Image } from 'theme-ui'
```

Comment:
@seri if we are going to use Image here, we need to create our own `Image` component so that we can do `<Image variant="avatar" />` and then just like Theme UI you can abstract that out further to provide an `<Avatar />` component that renders `<Image variant="avatar" />`. 

https://github.com/system-ui/theme-ui/blob/master/packages/components/src/Avatar.js
https://github.com/system-ui/theme-ui/blob/master/packages/components/src/Image.js

Reply:
@wesleycole should this not support children or overriding of the Image Child?  What if the developer wants to use a compiled svg instead of an image file or whats to use a letter when an image was not available?

Reply:
So there are a couple of reason why we want to only support images with `Avatar` and `Image`. 

1. It's semantic. An avatar and Image are typically only images, not rendered SVGs or whatever is passed as a child
2. Rendering a single character or multiple characters within a circle often comes with a lot of CSS issues to center it perfectly with line-height, different fonts, font sizes, etc. There is no guarantee that we will know what the child will be or the text will be and thus will often cause the developer to have to overwrite more styles to get what they want. 
3. Rendering a single character within a circle, if an image is not present, should be done via SVG instead of an Image based component
4. Ensuring that only an Image is accepted allows us to utilize pure HTML attributes instead of creating our own custom props like `imageSrc` or `text`. Instead we just accept `src`, and `alt` as props and everything is taken care of for us and it is not a new API that the developer has to learn. They know that they are working with an image and thus they don't have to read the API documentation to know what props are supported. 

This is why I am proposing that `Avatar` be based off of our own internal `Image` component from Kodiak UI that accepts only an image. If a developer wants to render an icon with a single character in it or different text then we need to create a `Svg` component that will dynamically generate the svg for the developer based on the children passed in. 

That way we will perform a check where the avatar is rendered like: `{shop?.logo ? <Avatar /> : <Svg />}`. While `Avatar` is not as flexible of a component it makes things more composable because it actually gives the developer more flexibility to style and create dynamic avatars as well if needed rather than having to add props and coerce the Avatar component to display the appropriate text as needed.